packages/matchbox/src/components/Expandable/Expandable.js

```diff
@@ -48,56 +84,69 @@ function Expandable(props) {
     })(e);
   }
 
-  const iconMarkup = icon
-    ? <div className={styles.Icon}>{icon}</div>
-    : null;
+  const accentColor = accent === true ? 'blue' : accent;
 
-  const contentSpacer = icon
-    ? <div className={styles.ContentSpacer} />
-    : null;
+  const accentMarkup = accentColor ? <Accent accentColor={accentColor} /> : null;
 
-  const contentClasses = classnames(styles.ContentWrapper, isOpen && styles.Open);
-  const arrowClasses = classnames(styles.Arrow, isOpen && styles.Open);
+  const iconMarkup = icon ? (
+    <StyledIcon flex="0" minWidth="40px" maxWidth="40px" mr="500">
+      {icon}
+    </StyledIcon>
+  ) : null;
+
+  const subtitleMarkup = subtitle ? <StyledSubtitle>{subtitle}</StyledSubtitle> : null;
+
+  const contentSpacer = icon ? <Box flex="0" minWidth="40px" maxWidth="40px" mr="500" /> : null;
 
   return (
-    <div className={styles.Expandable}>
-      <div
-        aria-controls={id}
-        aria-expanded={isOpen}
-        className={styles.Header}
-        onClick={handleClick}
-        onKeyDown={handleKeyDown}
-        ref={header}
-        role='button'
-        tabIndex='1'
-        data-id="expandable-toggle"
-      >
-        {iconMarkup}
-        <div className={styles.HeaderContent}>
-          <h3 className={styles.Title}>{title}</h3>
-          <h6 className={styles.Subtitle}>{subtitle}</h6>
-        </div>
-        <div className={styles.HeaderArrow}>
-          <KeyboardArrowRight size={36} className={arrowClasses} />
-        </div>
-      </div>
-      <div
-        aria-hidden={!isOpen}
-        className={contentClasses}
-        id={id}
-        data-id="expandable-content"
-      >
-        {contentSpacer}
-        <div className={styles.Content}>
-          {children}
-        </div>
-      </div>
-    </div>
+    <Box {...rest}>
+      {accentMarkup}
+      <StyledExpandable accent={accent}>
+        <StyledHeader
+          aria-controls={id}
+          aria-expanded={isOpen}
+          onClick={handleClick}
+          onKeyDown={handleKeyDown}
+          ref={header}
+          role="button"
+          tabIndex="0"
```

Comment:
Now that you are using a native `<button>`, no need to add these attributes - they are already baked in, fortunately.